README.md

```diff
@@ -159,6 +159,65 @@ You'll need to upload these to an s3 bucket, then you can use these files to
 [create a simulation application](https://docs.aws.amazon.com/robomaker/latest/dg/create-simulation-application.html),
 and [create a simulation job](https://docs.aws.amazon.com/robomaker/latest/dg/create-simulation-job.html) in RoboMaker.
 
+
+## Generate map for your world
+
+Procedurally generate an occupancy map for any gazebo world. This map can then be plugged into your navigation stack to navigate a robot in your world. 
+
+Note: This is an OPTIONAL functionality and is NOT required for using any functionalities listed above. (requires ROS-Melodic + Gazebo9)
```

Comment:
Why does it only work for Melodic/Gazebo 9? Is that a limitation of the plugin?

Reply:
This part of the README needs to be updated sorry.

Yes it was a limitation of the plugin.. however now I have updated the plugin to build and run with Gazebo7+Kinetic and Gazebo9+Kinetic and Gazebo9+Melodic..